Sources/PrimerSDK/Classes/Core/Payment Services/ClientTokenService.swift

```diff
@@ -7,8 +7,8 @@ internal typealias RawJWTToken = String
 
 internal protocol ClientTokenServiceProtocol {
     static var decodedClientToken: DecodedClientToken? { get }
-    static func storeClientToken(_ clientToken: String) -> Promise<Void>
-    static func storeClientToken(_ clientToken: String, completion: @escaping (Error?) -> Void)
+    static func storeClientToken(_ clientToken: String, isAPIValidationEnabled: Bool) -> Promise<Void>
+    static func resetClientToken()
```

Comment:
Just checking in...
I've seen we added this `resetClientToken()` though I've never seen it implemented throughout the changes

Reply:
Because it was already there, just not in the protocol